src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write your code here
+const population = [...document.querySelectorAll('.population')];
+const totalP = document.querySelector('.total-population');
+const averageP = document.querySelector('.average-population');
+
+const populationNumbers = [];
+
+population.forEach((span) => {
+  const populationString = span.textContent;
+  const cleanedString = populationString.replace(/,/g, '');
+  const numberOfPopulation = parseInt(cleanedString, 10);
+
+  if (!isNaN(numberOfPopulation)) {
+    populationNumbers.push(numberOfPopulation);
+  }
+});
+
+const totalPopulation = populationNumbers.reduce((acc, num) => acc + num, 0);
+
+const averagePopulation = totalPopulation / populationNumbers.length;
```

Comment:
There is a potential division by zero error here. If `populationNumbers` is empty, `populationNumbers.length` will be zero, leading to a division by zero when calculating `averagePopulation`. Consider adding a check to ensure `populationNumbers` is not empty before performing the division.